CharlieBackend.Api.UnitTes/LessonServiceTests.cs

```diff
@@ -178,142 +278,58 @@ public async Task CreateLessonAsync()
                     Presence = false
```

Comment:
> StudentId = 1

Please replace the magic number in the line above with a constant.

Reply:
done